documentation/src/main/asciidoc/topics/ref_rest_protobuf.adoc

```diff
@@ -37,6 +37,13 @@ GET /rest/v2/schemas/{schemaName}
 = Updating Protobuf Schemas
 Modify Protobuf schemas with `PUT` requests that include the content of a protobuf file in the payload.
 
+[IMPORTANT]
+====
+When you change existing Protobuf schema definition, you must reindex the data in the index.
+Rebuilding indexes has impact on performance and can take a long time to complete because the process takes place for all data in the grid.
+While the rebuild operation is in progress, queries might also return fewer results.
+====
+
```

Comment:
It is possible that the change can not have an impact on the indexes, e.g.: if the related caches are not indexed.
I would write something more general, such as:
> If the change has an impact on some indexed caches, these should be aligned if we want the change to be visible for them. To align the index schema is it possible to reindex (valid for the general case) or update the index schema (only if the changes do not touch pre-existing fields).